packages/react-components/src/components/OptionsButton.tsx

```diff
@@ -63,13 +64,18 @@ const generateDefaultMenuProps = (props: OptionsButtonProps): { items: Array<any
           title: val.name,
           canCheck: true,
           isChecked: val.id === selectedCamera?.id,
-          onClick: () => !(val.id === selectedCamera?.id) && onSelectCamera(val)
+          onClick: () => {
+            if (val.id !== selectedSpeaker?.id) {
```

Comment:
Should it be selectedCamera here and not selectedSpeaker?

Reply:
Another C/P error.. Thanks for spotting that.